apps/src/util/experiments.js

```diff
@@ -68,6 +68,8 @@ experiments.NEW_SIGN_UP_FLOW = 'new_sign_up_flow';
 experiments.VIEW_CHAT_HISTORY = 'view_chat_history';
 // Allows user to view the new version of the teacher navigation
 experiments.TEACHER_LOCAL_NAV_V2 = 'teacher-local-nav-v2';
+// Use the polling API for fetching chat responses in the AI Chat lab
+experiments.AI_CHAT_POLLING = 'ai-chat-polling';
```

Comment:
In your screencast, the experiment name you use is 'aichat-polling' and you refer to experiment as 'aichat-polling' in `aichatRedux`. Remove hyphen/underscore here for consistency?
How is this working now - does the hyphen not matter?

Reply:
Oh good catch 😅 this worked in the screencast because I didn't use this constant, and instead referenced the experiment name directly ("aichat-polling"). I'll update this constant and actually use it in experiment check.